copy.go

```diff
@@ -19,6 +22,24 @@ type Config struct {
 	Log         bool   `yaml:"log"`
 }
 
+func NewConfig(filePath string) (*Config, error) {
+	if err := utils.ValidateFilePath(filePath); err != nil {
+		return nil, err
+	}
+
+	file, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return nil, err
+	}
+
+	var c Config
+	if err = yaml.Unmarshal(file, &c); err != nil {
+		return nil, err
+	}
+
+	return &c, nil
+}
+
```

Comment:
I like what you did with NewConfig and ValidateFilePath. Perhaps this kind of functionality could be replicated with openSQL-related functions later on.

Reply:
(Btw please change the name of `openSQL` to `openMySQL`. There are going to be a few merge conflicts with unstable when the merge occurs I can handle resolving them.

Reply:
```openSQL``` has been renamed to ```openMySQL```